koku/api/report/test/azure/openshift/test_ocp_azure_query_handler.py

```diff
@@ -1399,9 +1381,10 @@ def test_ocp_azure_date_order_by_cost_desc(self):
             )
         correctlst = [service.get("service_name") for service in expected]
         for element in data:
-            lst = [service.get("service_name") for service in element.get("service_names", [])]
-            if lst and correctlst:
-                self.assertEqual(correctlst, lst)
+            if element.get("date") == str(yesterday):
```

Comment:
Why are we only checking yesterday now?

Reply:
👀  nice catch, this was from earlier triaging and has been reversed in this commit https://github.com/project-koku/koku/pull/3854/commits/726154c92b89452790d7e8230ef2739d9c6e7a05